src/object/game/arenasprite.lua

```diff
@@ -23,7 +23,7 @@ end
 
 function ArenaSprite:draw()
     if self.background then
-        love.graphics.setColor(0, 0, 0)
+        love.graphics.setColor(self.arena.bgColor[1], self.arena.bgColor[2], self.arena.bgColor[3])
```

Comment:
This function takes a table, so you can just do `love.graphics.setColor(self.arena.bgColor)`. That'd also allow it to support alpha, which this doesn't do